daprdocs/content/en/developing-applications/building-blocks/scheduler/scheduler-overview.md

```diff
@@ -0,0 +1,64 @@
+---
+type: docs
+title: "Scheduler overview"
+linkTitle: "Overview"
+weight: 1000
+description: "Overview of the scheduler API building block"
+---
+
+The Scheduler API works as an orchestrator for scheduling jobs in the future,  either at a specific time or a specific interval. The scheduler helps you with jobs like:
+
+- Scalable actor reminders
+- Scheduling any Dapr API to run at specific times or intervals, like when:
+  - Sending pub/sub messages
+  - Calling service invocations and input bindings
+  - Saving state to a state store 
+
+The Scheduler consists of two parts that work together to seamlessly schedule jobs across all of Dapr's API building blocks:
+- The Scheduler building block
+- [The Scheduler control plane service]({{< ref "concepts/dapr-services/scheduler.md" >}})
+
+<img src="/images/scheduler/scheduler-architecture.png" alt="Diagram showing basics of the Scheduler control plane and the Scheduler API">
+
+## How it works
+
+The Scheduler building block is a job orchestrator, not executor. The design guarantees *at least once* job execution with a bias towards durability and horizontal scaling over precision. This means:
+- **Guaranteed:** A job is never invoked *before* the schedule is due.
+- **Not guaranteed:** A ceiling time on when the job is invoked *after* the due time is reached.
+
+## Features
+
+### Delayed pub/sub
+
+The Scheduler building block enables you to delay your pub/sub messaging. You can publish a message in a future specific time -- for example, a week from today, or a specific UTC date/time.
+
+### Scheduled service invocation
+
+The Scheduler building block provides the [service invocation]({{< ref service-invocation-overview.md >}}) building block with an orchestrator that schedules method calls between applications.
+
+### Schedule jobs across multiple replicas
+
+The Scheduler service enables the scheduling of jobs to scale across multiple replicas, while guaranteeing that a job is only triggered by 1 scheduler service instance.
+
+### Scheduler reminders
+
+Actors have actor reminders, but present some limitations involving scalability. Make reminders more scalable by using `ScheduleReminders`. 
+
+### Store job details separately from user-associated data
+
+If you'd like to store your user-associated data in a specific state store of your choosing, then you can:
+
+1. Provision a state store using the [Dapr state management building block]({{< ref howto-get-save-state.md >}}).
+1. Set `jobStateStore ` as `true` in the state store component’s metadata section. 
```

Comment:
We also don't have this quite yet, but will in a following release

Reply:
@cicoyle To clarify - should I save the whole "Store job details separately from user-associated data" section for a later release? Or just the second numbered item?

Reply:
Technically, we dont have support for the `jobStateStore` yet, and will not for 1.14. Everything will be stored in etcd atm. The `jobStateStore` feature will be added post 1.14.